tests/cephfs/cephfs_utilsV1.py

```diff
@@ -3928,3 +3928,18 @@ def parse_caps(self, output):
                 cap_value = parts[1].strip().strip('"')
                 caps[cap_type] = cap_value
         return caps
+
+    def enable_distributed_pin_on_subvolumes(
```

Comment:
We may also need to validate if pinning happens after enablement, reference - validate_pin_stats in https://github.com/red-hat-storage/cephci/blob/master/tests/cephfs/snapshot_clone/cg_snap_utils.py

Reply:
Sure. Will take this as enhancement